src/common/options/osd.yaml.in

```diff
@@ -600,7 +600,7 @@ options:
   desc: Duration before issuing a cluster-log warning
   long_desc: Waiting too long for a replica to respond (after at least half of the
     replicas have responded).
-  default: 2200
+  default: 22000
```

Comment:
How do we tackle the problem in `main`?
If this patch is truly reef-specific: doesn't a change of configurable's default in a minor release deserve a release note?